bindings/python/src/cloudsmith_api/models/package_tag_request.py

```diff
@@ -33,7 +33,7 @@ class PackageTagRequest(object):
                             and the value is json key in definition.
     """
     swagger_types = {
-        'action': 'int',
+        'action': 'str',
```

Comment:
Pretty sure this is as expected... right?

Reply:
Seems right, action is an enum. Looks like it lines up with other endpoints api spec (Vulnerability Policy)